.circleci/config.yml

```diff
@@ -63,9 +63,66 @@ jobs:
                 path: ~/project/dapp/junit
             - store_artifacts:
                 path: ~/project/dapp/junit
+    
+    getline_ts_build:
+        docker:
+            - image: getline/ci-frontend:latest
+        working_directory: ~/project/getline.ts
+        steps:
+            - checkout:
+                path: ~/project
+            - restore_cache:
+                keys:
+                    - getlinets2-modules-{{ checksum "yarn.lock" }}
+                    - getlinets2-modules
```

Comment:
Why restore a key you don't use/save?

Reply:
It is being used - cache keys in CircleCI are hierarchized using hyphens. So if we don't find our exact cache for modules, we grab another one that's similar and recent. This still reduces `yarn` time.